chart/apisix/values.yaml

```diff
@@ -95,34 +102,51 @@ admin:
     admin: edd1c9f034335f136f87ad84b625c8f1
     viewer: 4054f7cf07e344346cd3f287985e76a2
 
+allow:
+  # The ip range for allowing access to Apache APISIX
+  ipList:
+    - 127.0.0.1/24
+
 # APISIX plugins to be enabled
 plugins:
```

Comment:
I think we can delete `plugins`, use the default list should be enough

Reply:
I keep it here is to leave a configuration item(placeholder) for users who wanted to use plugins belong themself.
 In fact, we can indeed set the plug-in list to be empty, I made it compatible.
But I am not sure if we should set the plugin list empty by default?

Reply:
only comment should be enough if you want the user to know how to use it